README.md

```diff
@@ -15,8 +15,10 @@
 [![static analysis](https://github.com/yiisoft/rbac-rules-container/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/rbac-rules-container/actions?query=workflow%3A%22static+analysis%22)
 [![type-coverage](https://shepherd.dev/github/yiisoft/rbac-rules-container/coverage.svg)](https://shepherd.dev/github/yiisoft/rbac-rules-container)
 
-The package provides rules container for [Yii RBAC (Role-Based Access Control)](https://github.com/yiisoft/rbac) package
-based on [Yii Factory](https://github.com/yiisoft/factory).
+This package is a factory for creating [Yii RBAC (Role-Based Access Control)](https://github.com/yiisoft/rbac) rules. It 
+provides rules container wrapping [Yii Factory](https://github.com/yiisoft/factory) and uses 
+[Yii Definitions](https://github.com/yiisoft/definitions) syntax. RBAC manager considers rule as a name and parameters
+and unaware of its creation by design, delegating creation to rules container keeping responsibilities separation.
```

Comment:
```suggestion
This package is a factory for creating [Yii RBAC (Role-Based Access Control)](https://github.com/yiisoft/rbac) rules. It 
provides rules container based on [Yii Factory](https://github.com/yiisoft/factory) and uses 
[Yii Definitions](https://github.com/yiisoft/definitions) syntax. RBAC manager accepts rule as a name and parameters
and is unaware of its creation by design, delegating creation to rules container keeping responsibilities separation.
```

Reply:
Fun fact about "based on" - that's how it was initially. :slightly_smiling_face: